src/components/dataset-card.tsx

```diff
@@ -0,0 +1,26 @@
+import {Dataset} from '@/lib/dataset-fetcher';
+import Link from 'next/link';
+
+interface DatasetCardProps {
```

Comment:
Or change to the generic `Props`, for readability of type and function signature?